### PR TITLE
Add playbook to support lvm

### PR DESCRIPTION
**Pull request by msambol**
This creates a logical volume across several physical devices.
The lv is then mounted, say, at `/var/lib/nova/instances`.
This will allow us to later extend the volume without affecting 
the mount point. 

Example usage:
`ursula envs/example/lvm playbooks/lvm.yml`

### CODE REVIEW

**Comment by j2sol** [comment]
So the assumption is that what Ursula gets handed is a pile of machines where a bunch of empty space is available? And then we'd boot strap them for LVMs before doing the convergence run? Seems reasonable, but might want to bundle it into the existing `playbooks/bootstrap_....` files, maybe as an include set of tasks/playbook. Might want to loop @sandfish8 in on this for how the workflow should work when implementations stands up a cloud.

File: playbooks/lvm.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+- name: lvm
```
Comment (j2sol) [inline playbooks/lvm.yml]: I think we'd probably want to use group vars or host vars to define the expected LVM bits, like `compute` group gets the `/var/lib/nova/instances` path, while others would get.... something.

File: playbooks/lvm.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      lvg:
+        vg: "{{ lvm.volume_group }}"
+        pvs: "{{ lvm.physical_volumes }}"
+        pesize: "{{ lvm.physical_extent_size }}"
+
+    - name: create logical volume
+      lvol:
+        vg: "{{ lvm.volume_group }}"
+        lv: "{{ lvm.logical_volume }}"
+        size: "{{ lvm.size }}"
```
Comment (j2sol) [inline playbooks/lvm.yml]: Should this maybe loop over a hash of LVs to create, so a system can have more than one easily defined in the vars?

File: playbooks/lvm.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+    - name: create filesystem
+      filesystem:
+        fstype: "{{ lvm.filesystem }}"
+        dev: "/dev/{{ lvm.volume_group }}/{{ lvm.logical_volume }}"
```
Comment (j2sol) [inline playbooks/lvm.yml]: Same here, loop over the filesystems to create.

File: playbooks/lvm.yml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+    # this also configures device in /etc/fstab!
+    - name: mount filesystem
+      mount:
+        name: "{{ lvm.mount_point }}"
+        src: "/dev/{{ lvm.volume_group }}/{{ lvm.logical_volume }}"
+        fstype: "{{ lvm.filesystem }}"
+        state: mounted
```
Comment (j2sol) [inline playbooks/lvm.yml]: and here.

**Comment by msambol** [comment]
@j2sol Good comments :+1: 
I think we need more discussion around this before I continue....

**Pull request closed without merging**